src/main/scala/firrtl/Emitter.scala

```diff
@@ -15,8 +14,9 @@ import firrtl.PrimOps._
 import firrtl.WrappedExpression._
 import Utils._
 import MemPortUtils.{memPortField, memType}
-import firrtl.options.{Dependency, HasShellOptions, ShellOption, StageUtils, PhaseException, Unserializable}
+import firrtl.options.{Dependency, HasShellOptions, PhaseException, ShellOption, StageUtils, Unserializable}
```

Comment:
this is annoying thing that IntelliJ does. However, while unrelated to this PR, I think we should just keep it this way so that future edits to the Emitter don't have to manually undo IntelliJ's import sorting.